src/odemis/model/_futures.py

```diff
@@ -1,19 +1,26 @@
+#!/usr/bin/env python3
```

Comment:
I think this one is not necessary here, you just need it at the top of the test cases.